src/arsort.py

```diff
@@ -663,3 +662,24 @@ def make_dirs(argflag, fitsdict, filesort):
     else: os.mkdir(newdir)
     # Return the name of the science targets
     return sci_targs
+
+
+def set_instr_name(inst_name):
```

Comment:
This function can be dropped now, right? I presume it has been deleted in your code, but is still showing up here because of the previous version of the PR? Anyway, just checking.